packages/playwright-core/src/server/dispatchers/playwrightDispatcher.ts

```diff
@@ -47,11 +48,12 @@ export class PlaywrightDispatcher extends Dispatcher<Playwright, channels.Playwr
   private _browserDispatcher: BrowserDispatcher | undefined;
 
   constructor(scope: RootDispatcher, playwright: Playwright, options: PlaywrightDispatcherOptions = {}) {
-    const chromium = new BrowserTypeDispatcher(scope, playwright.chromium);
-    const firefox = new BrowserTypeDispatcher(scope, playwright.firefox);
-    const webkit = new BrowserTypeDispatcher(scope, playwright.webkit);
-    const bidiChromium = new BrowserTypeDispatcher(scope, playwright.bidiChromium);
-    const bidiFirefox = new BrowserTypeDispatcher(scope, playwright.bidiFirefox);
+    const denyLaunch = options.denyLaunch ?? false;
```

Comment:
I just now thought that we should prevent electron and android launches as well, just in case.